core/src/commonMain/kotlin/com/ustadmobile/core/domain/credentials/passkey/webAuthn/PasskeyWebAuthNResponse.kt

```diff
@@ -0,0 +1,15 @@
+package com.ustadmobile.core.domain.credentials.passkey.webAuthn
+
+import kotlinx.serialization.Serializable
+
```

Comment:
This needs a comment with a link to the specification; this is a model based on an external specification.